### PR TITLE
Major Fix: [Moved admin related Controllers to a separate subdir/namespace, structured admin route, complete expense crud]

### PR DESCRIPTION
**Pull request by AironDev**
image
The admin routes were still very very disorganized, so I have been able to put them all together

image
ADMIN specific controllers have been moved to a subdir/namespace

image
session flash template wich can @included in admin pages

### CODE REVIEW

**Comment by goodhands** [comment]
Nice job here bro. Kindly update the team on the new changes and I trust you have tested and all works fine?

File: app/Http/Controllers/Admin/CompanyController.php [hunk 1]
```diff
@@ -31,14 +31,14 @@ public function viewCompanies()
         return view('backend.company.view')->with(['companies' => $companies]);
     }
 
-     public function createCompany(Request $request)
+    public function createCompany(Request $request)
     {
         $this->validate($request, [
```
Comment (goodhands) [inline app/Http/Controllers/Admin/CompanyController.php:36]: Why not just use `$request->validate()`? 
I am not certain `$this->validate()` exists

File: app/Http/Controllers/Admin/ExpenseController.php [hunk 2]
```diff
@@ -23,7 +22,7 @@ public function createExpense(Request $request)
          return view('backend.expense.create');
```
Comment (goodhands) [inline app/Http/Controllers/Admin/ExpenseController.php:22]: Are we returning 2 views here?

File: app/Http/Controllers/Admin/MinistryController.php [hunk 3]
```diff
@@ -105,7 +105,8 @@ public function editMinistry(Request $request, $id)
         }
     }
 
-    public function deleteMinistry($id){
+    public function deleteMinistry($id)
+    {
         $delete = Ministry::where('id', $id)->delete();
         if ($delete) {
             echo ("<script>alert(' Ministry details deleted successfully');
```
Comment (goodhands) [inline app/Http/Controllers/Admin/MinistryController.php:112]: Whoever wrote this code ehn, make them comot this script

**Review by goodhands — APPROVED** [review]
Nice job bro.

**Comment by goodhands** [comment]
Travis CI failed, see a screenshot of the error here.
image

File: app/Http/Controllers/Admin/CompanyController.php (hunk 1 above)
Comment (yanmifeakeju) [reply app/Http/Controllers/Admin/CompanyController.php:36]: It exists.

File: app/Http/Controllers/Admin/ExpenseController.php (hunk 2 above)
Comment (AironDev) [reply app/Http/Controllers/Admin/ExpenseController.php:22]: view all expenses, create new expense

File: app/Http/Controllers/Admin/MinistryController.php (hunk 3 above)
Comment (AironDev) [reply app/Http/Controllers/Admin/MinistryController.php:112]: he will -

**Comment by AironDev** [comment]
I have added another commit to fix the travis CI build error, I needed to delete the initial AdminController file.
Everything works perfectly on local

**Pull request merged**